docs/src/troubleshooting.md

```diff
@@ -61,43 +68,29 @@ Another solution could be (at least on Linux) to update the server `PATH` using
 
 ### 1. LSP doesn't start my language server
 
-* Make sure you have a folder added in your Sublime workspace.
-* Make sure the document you are opening lives under that folder.
-
-Your client configuration requires two settings to match the document your are editing:
+When language server is started, its name appears on the left side of the status bar. If you expect your server to start for a particular file but it doesn't then:
 
-* Scope (eg. `source.php`): Verify this is correct by running "Show Scope Name" from the developer menu.
-* Syntax (eg. `Packages\PHP\PHP.sublime-syntax`): Verify by running `view.settings().get("syntax")` in the console.
+* Make sure that the root scope (eg. `source.php`) of the file matches the scope handled by the language server. You can check the root scope of the file by running `Show Scope Name` from the `Tools -> Developer` menu. Refer to the documentation of the language server or its own settings to know the expected scope.
+* Make sure that the language server is not disabled globally either in its own settings or in `Preferences: LSP Settings`, or in the project settings (`Project: Edit Project` from the Command Palette).
 
 ### 2. LSP cannot find my language server (`No such file or directory: 'xyz'`)
-Assuming that the server is actually installed, and that you can start it from your shell, this issue is likely due to Sublime Text's internal environment not picking up the same `PATH` environment variable as you've configured in your shell.
 
-The exact changes to make can differ depending on what program you want to expose to Sublime Text. The simplest way is to extend the path like so (replacing `/usr/local/bin` with the path of your choice):
-```sh
-export PATH="/usr/local/bin:$PATH"
-```
-
-If, for example, you want to expose a `Node` binary to ST and you have it installed through a version manager like `nvm`, you need to insert its [initialization script](https://github.com/nvm-sh/nvm#install--update-script) in the location specified in [this table](troubleshooting.md#updating-the-path-used-by-lsp-servers)
```

Comment:
This part about "nvm" moved to the "Updating the PATH used by LSP servers" section